utils/ci-status.py

```
@@ -0,0 +1,615 @@
+        return f"{COLORS[status]}{text}{RESET}"
+    return text
+
+
+def http_json(url, token=None, timeout=30):
+    headers = {
+        "Accept": "application/json",
+        "User-Agent": "postgis-ci-status/1.0",
+    }
+    if token:
+        headers["Authorization"] = f"Bearer {token}"
+    request = urllib.request.Request(url, headers=headers)
```

Ruff (0.15.18)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)